### ISSUE TITLE
Auto-indent doesn't work within multi-line C-style comments

### ISSUE THREAD

**Issue by danpla** [geany/geany#2080]
The auto-indent doesn't work within C-style multi-line comments: it ignores the indentation of the previous line. For example (`|` is caret):
```
/**
 * Multi line comment.
 *     Indented line|
 */
```
The current indentation after pressing Enter:
```
/**
 * Multi line comment.
 *     Indented line
 * |
 */
```
The expected indentation:
```
/**
 * Multi line comment.
 *     Indented line
 *     |
 */
```

**Comment by danpla** [comment]
In case this is an expected behavior rather than a bug, please consider it as a feature request.

**Comment by elextr** [comment]
Yes its the expected behaviour.  Indents apply before the first non-whitespace character only, so not after the `*`.
